From de047e6f4ce661841d4a5a8934c47f97de9296c2 Mon Sep 17 00:00:00 2001
From: Lucas Pluta <lp.whs86@gmaii.com>
Date: Fri, 5 Nov 2021 00:45:16 -0400
Subject: [PATCH] Patch to remove github hooks

---
 src/Directory.Build.props | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/Directory.Build.props b/src/Directory.Build.props
index 93251be..587e8e0 100644
--- a/src/Directory.Build.props
+++ b/src/Directory.Build.props
@@ -5,8 +5,6 @@
     <IsPackable>true</IsPackable>
   </PropertyGroup>
   <ItemGroup>  
-    <PackageReference Include="Nerdbank.GitVersioning" Version="3.3.37" PrivateAssets="all" Condition=" '$(DEVCONTAINER)' != 'true' " />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="all"/>
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" PrivateAssets="All" Version="1.0.0" />
   </ItemGroup>
 </Project>
-- 
2.30.2.windows.1

