From c2bba6342c75f810e69e0bc229329984a5437b4b Mon Sep 17 00:00:00 2001
From: Lucas Pluta <lp.whs86@gmaii.com>
Date: Fri, 12 Nov 2021 15:20:22 -0500
Subject: [PATCH] Patch to adjust the dotnet SDK version

---
 Src/global.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Src/global.json b/Src/global.json
index 6d19c9f..a0ee6a6 100644
--- a/Src/global.json
+++ b/Src/global.json
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "5.0.200"
+    "version": "5.0"
   }
 }
\ No newline at end of file
-- 
2.30.2.windows.1

